README.md

```diff
@@ -46,6 +46,21 @@ once the init command has been launched:
 
 In order to build the product, just run `yarn` at the root folder (where theia has been cloned)
 
+### Custom extension set
+
+Also you can provide custom `yaml` with your extension set, by using `-c` or `--config` parameter of `che:theia init` :
+
+`che:theia init -c ./path/to/custom/extensions.yaml`
+
+The sample of `extensions.yaml` can be found [there](./src/conf/extensions.yml)
```

Comment:
I add the comment there but it doesn't belong to the README
Maybe we should remove the extensions.yml of the generator ? as it will be out of sync.

or add an option to create a new one to a directory but it may be useless as there are not so many users :-)

Reply:
I also think about removing `extensions.yml`
For generating new configuration we still need to have list of extensions somewhere

Reply:
maybe grab it from github (from che-theia repo ?)

Reply:
OK, that good

Reply:
it means `extensions.yml` is only at a single place so no sync issue.